tests/components/vacuum/test_demo.py

```diff
@@ -158,12 +158,12 @@ def test_methods(self):
         self.assertIn("spot", state.attributes.get(ATTR_STATUS))
         self.assertEqual(STATE_ON, state.state)
 
-        vacuum.start_pause(self.hass, ENTITY_VACUUM_STATE)
```

Comment:
Maybe I should have commented here instead...

Reply:
Yes it is covered by: https://github.com/cnrd/home-assistant/blob/5a6b52779dc3b9fc08bc4fe5b095fcd2cdaf2b4e/tests/components/vacuum/test_demo.py#L123-L129 and https://github.com/cnrd/home-assistant/blob/5a6b52779dc3b9fc08bc4fe5b095fcd2cdaf2b4e/tests/components/vacuum/test_demo.py#L223-L225